package.json

```diff
@@ -1,52 +1,53 @@
 {
   "name": "@react-three/babel",
-  "version": "0.1.1",
-  "main": "./dist/cjs/index.js",
-  "module": "./dist/esm/index.js",
-  "types": "./dist/esm/index.d.ts",
-  "files": [
-    "dist/"
-  ],
+  "version": "0.2.0",
```

Comment:
Should this be v1 instead? There's hardly any implementation here.

Reply:
Yeah, I was holding back on the v1 until we had this installed in more repos to test the robustness. But the scope of the plugin is so small that you might say it's feature complete at this point

Reply:
That's fair. I need to do integration testing with this, but I comment since this PR is a breaking change. So, I've bumped to 0.2 for 0.x semver, but it could also be v1 if the API is okay. Behavior changes are breaking also.